js/app.js

```diff
@@ -147,13 +147,35 @@ function addNewStore(event){
   event.preventDefault();
   var formEl = ['formName', 'formMinCustPerHour', 'formMaxCustPerHour', 'formAvgCookiesSales'];
   var objEl = [];
-  for (var i = 0; i < formEl.length; i++){
-    var x = event.target[formEl[i]].value;
+  // for (var i = 0; i < formEl.length; i++){
+  for(var i in formEl){
+    var x = (event.target[formEl[i]].value);
+    console.log(typeof(x));
     objEl.push(x);
   }
 
+  for (var z = 0; z < stores.length; z++){
+    if(stores[z].name === objEl[0]){
+      //do change data
+      console.log('if statement for name matching');
+      stores[z].minCustPerHour = parseInt(objEl[1]);
+      stores[z].maxCustPerHour = parseInt(objEl[2]);
+      stores[z].avgCookieSales = parseInt(objEl[3]);
+      console.log('obj ' + objEl[0]);
+      break ;
+    }
+    else{
+      //create a new store object
+      console.log('creating an if object');
+      new Store(objEl[0], parseInt(objEl[1]), parseInt(objEl[2]), parseInt(objEl[3]));
+      console.log(objEl[0]);
+      console.log(stores);
+      break;
+    }
```

Comment:
Since you have a `break` in both the `if` and `else` blocks, this loop will not actually loop. It will check the first `Store` object to see if the names match, and then stop the loop.